recipes-extended/sysint/sysint_git.bb

```diff
@@ -145,6 +145,16 @@ do_install() {
         install -m 0644 ${S}/systemd_units/zram.service ${D}${systemd_unitdir}/system
 
 
+	install -m 0644 ${S}/systemd_units/network-up.path ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/network-up.target ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/network-up.timer ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/ntp-sync.path ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/ntp-sync.target ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/ntp-sync-dbus-event.service ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/ntp-sync.timer ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/systime-set.path ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/systime-set.target ${D}${systemd_unitdir}/system
+	install -m 0644 ${S}/systemd_units/systime-dbus-event.service ${D}${systemd_unitdir}/system
```

Comment:
Rename to `broadcast-system-time-set.service`

Reply:
change to system-time-event.service